deploy/examples/secure-query-with-keycloak-sidecar-values.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Jaeger
+  metadata:
+    name: jaeger
+    namespace: default
+  spec:
+    agent:
```

Comment:
You can keep the example simpler if you remove the empty nodes (agent, allInOne, collector, ingester).